internal/restic/lock.go

```diff
@@ -264,7 +264,7 @@ func LoadLock(ctx context.Context, repo Repository, id ID) (*Lock, error) {
 }
 
 // RemoveStaleLocks deletes all locks detected as stale from the repository.
-func RemoveStaleLocks(ctx context.Context, repo Repository) error {
+func RemoveStaleLocks(ctx context.Context, repo Repository, processed *uint) error {
```

Comment:
Please change the method signature to `func RemoveStaleLocks(ctx context.Context, repo Repository) (uint, error)`. Go can return multiple values, so let's use that.

The same also applies to `RemoveAllLocks`.